pyscriptjs/src/exceptions.ts

```diff
@@ -31,13 +31,15 @@ export enum ErrorCode {
 export class UserError extends Error {
     messageType: MessageType;
     errorCode: ErrorCode;
+    $$isUserError: boolean;
```

Comment:
> The issue is that isinstance doesn't work correctly across multiple realms.

ok, makes sense and it's good to know. So basically, the logic works because `UserError` sets `$$userError = true`, and on all other exceptions we read `undefined`? It's not super elegant but I can live with that.

> $$ is a common way to indicate a marker.

what is a "marker" here? Is it a technical term with a specific meaning or just a synonym for "flag"?

Reply:
added a comment explaining this.